docs/configuration/arguments.md

```diff
@@ -404,7 +404,7 @@ It also talks to a KVStore and has it's own copies of the same flags used by the
 
 Cortex has a concept of "runtime config" file, which is simply a file that is reloaded while Cortex is running. It is used by some Cortex components to allow operator to change some aspects of Cortex configuration without restarting it. File is specified by using `-runtime-config.file=<filename>` flag and reload period (which defaults to 10 seconds) can be changed by `-runtime-config.reload-period=<duration>` flag. Previously this mechanism was only used by limits overrides, and flags were called `-limits.per-user-override-config=<filename>` and `-limits.per-user-override-period=10s` respectively. These are still used, if `-runtime-config.file=<filename>` is not specified.
 
-At the moment, two components use runtime configuration: limits and multi KV store.
+At the moment, three components use runtime configuration: per-user limits, multi KV store, and ingester instance limits.
```

Comment:
Shall we reword this to avoid "components"? For me, "component" means ingester or compactor.

What about: "At the moment runtime configuration may contain per-user limits, configuration for multi KV store and ingester instance limits."

Reply:
Good idea, will do!